Code/src/main/java/nl/utwente/viskell/ui/components/SliderBlock.java

```diff
@@ -11,14 +11,14 @@
  * Ranges from 0 to 1 (both inclusive).
  */
 public class SliderBlock extends ValueBlock {
-    @FXML private Slider slider;
+    @FXML protected Slider slider;
 
     /**
      * Constructs a new SliderBlock
      * @param pane The parent pane this Block resides on.
      */
-    public SliderBlock(CustomUIPane pane) {
-        super(pane, Type.con("Float"), "0.0", "SliderBlock");
+    public SliderBlock(CustomUIPane pane, Environment env) {
+        super(pane, env.buildType("Fractional a => a"), "0.0", "SliderBlock");
```

Comment:
Why pass `env` as an argument when you can just as easily do `pane.getEnvInstance` here?